php/Admin/EditBlock.php

```diff
@@ -136,14 +136,23 @@ public function enqueue_assets() {
 			'before'
 		);
 
-		$edit_block_style_path = 'css/dist/edit-block.css';
-		$css_config            = require $this->plugin->get_path( 'css/dist/edit-block.asset.php' );
+		$edit_block_style_path   = 'css/dist/edit-block.css';
+		$edit_block_style_config = require $this->plugin->get_path( 'css/dist/edit-block.asset.php' );
 		wp_enqueue_style(
 			self::STYLE_SLUG,
 			$this->plugin->get_url( $edit_block_style_path ),
 			[ 'wp-components' ],
-			$css_config['version']
+			$edit_block_style_config['version']
 		);
+
+		$editor_style_config = require $this->plugin->get_path( 'css/dist/blocks.editor.asset.php' );
```

Comment:
Enqueues the block editor styling in the GCB editor:
<img width="918" alt="block-editor-styling" src="https://user-images.githubusercontent.com/4063887/108116044-a3127800-7060-11eb-9200-99bfcff54c02.png">